openreview/agora/agora.py

```diff
@@ -1,5 +1,4 @@
-from .. import openreview
-
+import openreview
```

Comment:
does it mean that openreview has to be installed with pip and this import is linked to the local file in the repo folder?

Reply:
This means that instead of importing the local module openreview (coming from openreview.py), I am now importing the root level package openreview. 
This package would be actively linked to the local files if you installed the package in develop mode.

Reply:
I have reverted this change @melisabok